### ISSUE TITLE
UNREACHABLE executed at clspv/lib/ReplacePointerBitcastPass.cpp:834!

### ISSUE THREAD

**Issue by kpet** [google/clspv#9]
A few CL files from the [ARM Compute Library](https://github.com/ARM-software/ComputeLibrary) exhibit the following issue:

```
  %26 = bitcast i8 addrspace(1)* %call52 to i16 addrspace(1)*Handle above user of scalar bitcast with gep on ReplacePointerBitcastPass
UNREACHABLE executed at clspv/lib/ReplacePointerBitcastPass.cpp:834!
Aborted (core dumped)
```

Here's a couple that show the problem (just passing the file to `clspv`):
- [Optical flow](https://github.com/ARM-software/ComputeLibrary/blob/db0e610fa90a4c7c534049792ee7fe143d46f72f/src/core/CL/cl_kernels/optical_flow_pyramid_lk.cl)
- [Batch normalization](https://github.com/ARM-software/ComputeLibrary/blob/db0e610fa90a4c7c534049792ee7fe143d46f72f/src/core/CL/cl_kernels/batchnormalization_layer.cl)

**Comment by rjodinchr** [comment]
I would expect this to be fixed. @kpet can you confirm?

**Comment by kpet** [comment]
Likely yes. Both clspv and the Arm Compute Library will have changed enough over the past 5+ years that it does not make sense to keep this open. Closing.
